lib/soapy_bing/ads/bulk/campaigns.rb

```diff
@@ -4,15 +4,19 @@ class Ads
     module Bulk
       class Campaigns
         DEFAULT_ENTITIES = %w(CampaignTargets Ads).freeze
-        POLLING_TRIES = 30
+        DEFAULT_POLLING_SETTINGS = {
+          tries: 20,
+          sleep: ->(n) { n < 7 ? 2**n : 120 }
+        }.freeze
```

Comment:
Yea, I'm not saying we have to change it. Just wanted to make sure you're aware that this is a quite long timeframe. @hwartig thoughs?